python/mathsys/main/latex.py

```
@@ -48,30 +106,28 @@ def run(self, sheet: Level1) -> str:
+            match statement:
+                case Definition(): self.definition(statement)
+                case Declaration(): self.declaration(statement)
+                case Node(): self.node(statement)
+                case Equation(): self.equation(statement)
+                case Comment(): self.comment(statement)
+            self.latex.append(r"\\ ")
+        if len(sheet.statements) >= 1: self.latex.pop()
+        self.latex.append(delimiter)
+    # GENERATOR -> 2 DEFINITION GENERATION
+    def definition(self, definition: Definition) -> None:
+        self.latex.append(self.MAPPINGS[definition.identifier] if definition.identifier in self.MAPPINGS else definition.identifier)
```

Ruff (0.12.2)

Use `self.MAPPINGS.get(definition.identifier, definition.identifier)` instead of an `if` block

Replace with `self.MAPPINGS.get(definition.identifier, definition.identifier)`

(SIM401)